core/src/test/scala/com/rallyhealth/vapors/core/fql/PrinterSpec.scala

```diff
@@ -25,10 +25,10 @@ class PrinterSpec extends AnyWordSpec {
       }
     }
 
-    "print an expression that selects a fact type using where(factValue(_ > 200))" in {
+    "print an expression that selects a fact type using whereAnyFact(_.at(_.select(_.value)) > 200))" in {
       val q = {
-        __.withFactsOfType(FactTypes.WeightMeasurement)
-          .whereAnyValue(__ > 200)
+        __.withType(FactTypes.WeightMeasurement)
```

Comment:
With single underscore `_` now in use because of the `ConditionOps`, I'm not sure if I like the double underscore as much. Originally, it was meant to be a way to access all of the operations using auto-complete (including infix operators), but it isn't even doing that anymore (it only does the old-style infix operators with type selector methods).

Either I should go back to making `__` an alias to the full DSL (with or without infix operators), or I should just remove it and let the `dsl` package object be the starting point for the full DSL.